src/NHibernate/UserTypes/ICompositeUserType.cs

```diff
@@ -9,19 +9,22 @@ namespace NHibernate.UserTypes
 	/// A UserType that may be dereferenced in a query.
 	/// This interface allows a custom type to define "properties".
 	/// These need not necessarily correspond to physical .NET style properties.
-	///
-	/// A ICompositeUserType may be used in almost every way
+	/// </summary>
+	/// <remarks>
+	/// <para>
+	/// An <c>ICompositeUserType</c> may be used in almost every way
 	/// that a component may be used. It may even contain many-to-one
 	/// associations.
-	///
-	/// Implementors must be immutable and must declare a public
```

Comment:
Since there is a `IsMutable` property to implement, I have assumed this statement is obsolete.